src/components/editor-page/app-bar/sync-scroll-buttons/disabledScroll.svg

```diff
@@ -1,9 +1,63 @@
-<svg height="512" viewBox="0 0 135.46666 135.46666" width="512" xmlns="http://www.w3.org/2000/svg">
-    <g fill="currentColor" transform="translate(253.17277 890.86874)">
-        <path d="m-185.60701-890.86598c-1.20278.0405-2.35129.51236-3.23554 1.33104l-19.83931 18.35901c-2.0321 1.88196-2.16416 5.08233-.29441 7.12658 1.86949 2.04457 5.06645 2.17956 7.10004.29914l16.43678-15.21017 16.43678 15.21017c2.03361 1.88043 5.23055 1.74545 7.10004-.29914 1.86889-2.04514 1.73629-5.24555-.29677-7.12658l-19.83694-18.35901c-.96887-.89698-2.25281-1.37483-3.57067-1.33104z"/>
-        <path d="m-205.48644-783.84278c-1.28698.0542-2.55509.59901-3.48983 1.62131-1.86975 2.04426-1.73769 5.24463.29441 7.12659l19.83931 18.359c.88425.81868 2.03276 1.29062 3.23554 1.33104 1.31786.0439 2.6018-.43406 3.57067-1.33104l19.83694-18.359c2.03306-1.88102 2.16566-5.08144.29677-7.12659-1.86949-2.04459-5.06643-2.17956-7.10004-.29913l-16.43678 15.21017-16.43678-15.21017c-1.01679-.94021-2.32321-1.37647-3.61021-1.32218z"/>
-        <path d="m-185.49136-841.40292a18.267478 18.267478 0 0 0 -18.21562 18.26754 18.267478 18.267478 0 0 0 .23777 2.93583l22.52379-20.64169a18.267478 18.267478 0 0 0 -4.49402-.56168 18.267478 18.267478 0 0 0 -.0519 0zm18.08169 15.33288-22.5232 20.64052a18.267478 18.267478 0 0 0 4.49343.56168 18.267478 18.267478 0 0 0 18.26754-18.26754 18.267478 18.267478 0 0 0 -.23777-2.93466z"
-              fill-rule="evenodd"/>
-        <path d="m-124.79368-883.92129-126.14511 116.28534 4.95067 5.40084 126.14688-116.28534z"/>
-    </g>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  - SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+  -
+  - SPDX-License-Identifier: AGPL-3.0-only
+  -->
+
+<svg
+   height="16"
+   viewBox="0 0 16 16"
+   width="16"
+   version="1.1"
+   id="svg12"
+   sodipodi:docname="disabledScroll.svg"
+   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs16" />
+  <sodipodi:namedview
+     id="namedview14"
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1.0"
+     inkscape:pageshadow="2"
+     inkscape:pageopacity="0.0"
+     inkscape:pagecheckerboard="0"
+     showgrid="false"
+     fit-margin-top="0"
+     fit-margin-left="0"
+     fit-margin-right="0"
+     fit-margin-bottom="0"
+     scale-x="1"
+     inkscape:zoom="14.302332"
+     inkscape:cx="-21.814625"
+     inkscape:cy="-8.6699151"
+     inkscape:window-width="1141"
+     inkscape:window-height="1377"
+     inkscape:window-x="2826"
+     inkscape:window-y="28"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg12" />
```

Comment:
They get striped away in the result because svgr uses a SVG sanitizer.

Reply:
Yeah, but why do we include them in the git files at all?
Previously we haven't done that.

Reply:
The lib used by CRA doesn't sanitize the SVGs so I had to clean them up myself. That's not necessary anymore.

Reply:
So we want to push the uncleaned files in to the repo? I'm still not sure about this.
@DerMolly what's your opinion on this?

Reply:
Well. I don't see them as unclean. They just have the inkscape meta data. This data won't show up in the resulting Dom because the lib I use here pipes the file through svgo.

Imho it's not different from only including TS files into the repo and not the built JS.

Reply:
The difference is that TypeScript has the type annotations on purpose: For type checking.
I'm not sure why we would need to store the position of your inkscape window in the repository. It has literally no effect.
I mean we also don't store IDE settings in the Git repo (we excluded most of the `.idea` directory).

Reply:
I can see both sides of the argument and would like to point out that keeping the inkscape meta infos makes editing the files easier, but doesn't hurt the result. So I'm in favor of don't make @mrdrogdrog jump through additional hoops.

Reply:
In which sense does it make the editing easier? 🤔